packages/desktop-client/src/components/transactions/TransactionsTable.jsx

```diff
@@ -282,7 +282,17 @@ const TransactionHeader = memo(
             onSort('deposit', selectAscDesc(field, ascDesc, 'deposit', 'desc'))
           }
         />
-        {showBalance && <Cell value="Balance" width={88} textAlign="right" />}
+        {showBalance && (
+          <Cell
```

Comment:
Can we try using `HeaderCell` instead of `Cell` to see if that fixes the issue?

Reply:
I did that in my version of the fix, #3082.

Reply:
@joel-jeremy I was thinking of it but decided against it as the HeaderCell has some sorting functionality as well. And when clicking on the header it behaves wrongly for this field. That is why I kept the usage of the Cell component

Reply:
@Chuvisco88 good call. I updated my PR to make it not clickable.